packages/web3-rpc-providers/package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "web3-rpc-providers",
-	"version": "1.0.0-rc.0",
+	"version": "1.1.0-rc.0",
```

```suggestion
	"version": "1.0.0-rc.1",
```
This should be 1.0.0-rc.1 as we can do multiple RCs with breaking changes until 1.0.0 is stable or final to release.